validate-seqtools/main.nf

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env nextflow
+
+/*
+  Copyright (C) 2022,  icgc-argo
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+  Authors:
+    Edmund Su
+*/
+
+/********************************************************************/
+/* this block is auto-generated based on info from pkg.json where   */
+/* changes can be made if needed, do NOT modify this block manually */
+nextflow.enable.dsl = 2
+version = '0.1.0'  // package version
+
+container = [
+    'ghcr.io': 'ghcr.io/icgc-argo/argo-data-submission.validate-seqtools'
+]
+default_container_registry = 'ghcr.io'
+/********************************************************************/
+
+
+// universal params go here
+params.container_registry = ""
+params.container_version = ""
+params.container = ""
+
+params.cpus = 1
+params.mem = 1  // GB
+params.publish_dir = ""  // set to empty string will disable publishDir
+
+
+// tool specific parmas go here, add / change as needed
+params.json_file = ""
+params.skip_md5sum_check = 'NO_FILE'
```

Comment:
Is boolean type more proper for `skip_md5sum_check`? If we define it as string, any string will trigger skip? like `No skipping`?

Reply:
Switched to boolean.
https://github.com/icgc-argo/argo-data-submission/blob/2ed0573fbe4a1be55ec51b67a3e56d2359b139c6/validate-seqtools/main.nf#L48

https://github.com/icgc-argo/argo-data-submission/blob/2ed0573fbe4a1be55ec51b67a3e56d2359b139c6/validate-seqtools/main.nf#L68